recipes/libverto/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(libverto REQUIRED core CONFIG)
```

Comment:
```suggestion
find_package(libverto REQUIRED CONFIG)
```

Reply:
there is no core component (there is no config file upstream, so it makes little sense to ask for specific components in this test package), no sure to understand why it was there.